mne/io/base.py

```diff
@@ -779,8 +779,8 @@ def filter(self, l_freq, h_freq, picks=None, filter_length='10s',
             See mne.filter.construct_iir_filter for details. If iir_params
             is None and method="iir", 4th order Butterworth will be used.
         copy : bool
-            If ``True``, return a modified copy. If ``False``, modify
-            in-place and return the original object.
+            If ``True``, return the modified original object. If ``False``,
```

Comment:
I think `return a modified copy` was less ambiguous, but I like the second part better now. It seems like a minor detail but we don't want people to get hung up on this.